coffeecard/CoffeeCard.Library/Services/v2/ProductService.cs

```diff
@@ -57,6 +58,92 @@ public async Task<Product> GetProductAsync(int productId)
             return product;
         }
 
+        public async Task<ProductResponse> AddProduct(AddProductRequest newProduct, IEnumerable<UserGroup> allowedUserGroups)
+        {
+            var product = new Product()
+            {
+                Price = newProduct.Price,
+                Description = newProduct.Description,
+                Name = newProduct.Name,
+                NumberOfTickets = newProduct.NumberOfTickets,
+                ExperienceWorth = 0,
```

Comment:
I'm not keen on setting this to 0 in the repository. Either this should have a default value of 0 on the Controller/API level or it should be provided by the client

Reply:
@jonasanker Considering, we don't use ExperienceWorth for anything, and will likely remove it entirely in the future, why do we want to expose it in the API, and make the client dependent on it?

Reply:
Only thought is that by having this on the service/repository layer the behavior of the experiment worth is hidden and has a side effect (that it's always 0). I consider this a technical debt choice.

By giving it a default value on the API layer at least the client has a option to override it